### PR TITLE
fixing z-index issue with library hovercard over profile library cards

### PR DESCRIPTION
**Pull request by 2is10**
@yipingliao cc @ahsu1230

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl [hunk 1]
```diff
@@ -2,10 +2,7 @@
   position: fixed
   width: 270px
   padding: 0 0 0 20px
-  z-index: 1
-
-  .kf-tag-count
-    float: right
+  z-index: 2
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl:5]: The user picture in the user profile library cards also has `z-index: 1` so bumping this up to 2.

File: kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl [hunk 2]
```diff
@@ -2,10 +2,7 @@
   position: fixed
   width: 270px
   padding: 0 0 0 20px
-  z-index: 1
-
-  .kf-tag-count
-    float: right
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl:8]: As far as I can tell, the left nav no longer ever contains a `kf-tag-count`.

File: kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl [hunk 3]
```diff
@@ -20,14 +17,11 @@
...
+    transition-duration: sidebarTransitionDuration
 
-  .kf-sidebar-inactive .kf-sidebar
-    left: -135px
-    z-index: -1
+    .kf-sidebar-inactive &
+      transform: translate(-50%)
+      opacity: 0
+      visibility: hidden
+      transition-delay: 0, 0, sidebarTransitionDuration
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl:27]: `opacity` and `visibility` make for a better hiding transition than `z-index`.

**Comment by 2is10** [comment]
There’s still some funkiness where the first library heading jumps up/down when this sidebar transition is triggered. I don’t know why it has `fixed` position, but I have a feeling it’s a rabbit hole, so leaving alone for now.

**Comment by yipingliao** [comment]
:+1:

File: kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl (hunk 3 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/layout/sidebar/sidebar.styl:27]: :+1:

**Pull request merged**